packages/webchat/src/core/api.tsx

```diff
@@ -13,12 +13,14 @@ export default class WebchatApi {
 
   constructor(private socket: BpSocket) {}
 
+  // TODO: Do we still need this?
   private get baseUserPayload() {
     return {
       webSessionId: window.__BP_VISITOR_SOCKET_ID
     }
   }
 
+  // TODO: Fix this
   async fetchBotInfo() {
     try {
       const { data } = await this.axios.get('/botInfo', this.axiosConfig)
```

Comment:
This one will be done by fetching the botInfo.json file that will be located at https://mediafiles.botpress.dev/clientId/info.json